### PR TITLE
Query data and update tables

### PR DESCRIPTION
**Pull request by Trust-007**
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

Screenshot (50)
Screenshot (51)
Screenshot (42)
Screenshot (43)
Screenshot (44)
Screenshot (45)
Screenshot (46)
Screenshot (47)
Screenshot (48)
Screenshot (49)
Screenshot (52)
Screenshot (53)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -2,9 +2,37 @@
...
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > DATE '2022-01-01';
+SAVEPOINT FIRST_SAVE;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO FIRST_SAVE;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE SIGN(weight_kg) < 0;
+COMMIT;
+
```
Comment (mckent05) [inline queries.sql:33]: - Please include a query to perform this `How many animals are there?`

File: queries.sql [hunk 2]
```diff
@@ -2,9 +2,37 @@
...
+SELECT COUNT(age) FROM users;
+SELECT COUNT(name) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered,SUM(escape_attempts) FROM animals GROUP BY(neutered);
+SELECT MAX(weight_kg), MIN(weight_kg) FROM animals;
```
Comment (mckent05) [inline queries.sql]: - This query should get the minimum and max weight for each type of animal (GROUP_BY Species). Please fix this 🙏 
- Also please include a query to get `the average number of escape attempts per animal type of those born between 1990 and 2000?`

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Trust-007, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- Please don't forget to update your PR with a screenshot of the changes you made 🙏 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Trust-007 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**